Samples/Sample.Xamarin.Droid/MainActivity.cs

```diff
@@ -17,7 +17,7 @@ protected override void OnCreate(Bundle savedInstanceState)
         {
             SentryXamarin.Init(options =>
             {
-                options.Dsn = "https://5a193123a9b841bc8d8e42531e7242a1@o447951.ingest.sentry.io/5560112";
```

Comment:
https://github.com/mono/mono/issues/21233

The previous dsn had SSL issues where the dsn without the ingest works fine.